packages/cli/templates/default-plugin/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "@backstage/cli/config/tsconfig.json",
+  "include": [
+      "src",
+      "dev",
+      "migrations"
+  ],
+  "exclude": ["node_modules"],
+  "compilerOptions": {
+    "outDir": "dist-types",
+    "skipLibCheck": true
```

Comment:
Could skip this as it won't be needed after #2317

Reply:
ok